src/schemas/json/function.json

```diff
@@ -514,6 +516,59 @@
           "description": "The text of the email."
         }
       }
+    },
+    "sqlBinding": {
+      "properties": {
+        "connectionStringSetting": {
+          "type": "string",
+          "description": "The name of the app setting that contains the SQL connection string used to connect to the database."
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "type": {
+              "const": "sqlTrigger"
```

Comment:
Curious, why it should be `const` here, while all other bindings are using enum like below -

```json
 "properties": {
            "type": {
              "enum": ["blobTrigger"]
            },
```

Reply:
They're both essentially the same in this scenario. The others should probably be using const TBH - as enum implies there's multiple potential values. In our case (and theirs too) there's only one possible value so const seems more appropriate.